Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Application/Controllers/SecurityTestController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Kros.AspNetCore.Authorization;
+
+namespace Kros.CqrsTemplate.Application.Controllers
+{
+    /// <summary>
+    /// SecurityTest controller.
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
```

Comment:
By my opinion status codes `404` and `500` are misconceptions.
I would replace them with `401 - Unauthorize`

Reply:
Aggreed. Fixed in next commit.